src/components/DropZone/DropZone.tsx

```diff
@@ -327,30 +330,36 @@ export class DropZone extends React.Component<CombinedProps, State> {
 
     this.setState({error: this.props.error});
 
-    if (!this.dropNode) {
+    const dropNode = this.dropNode.current || document;
+
+    if (!dropNode) {
       return;
     }
 
-    addEventListener(this.dropNode, 'drop', this.handleDrop);
-    addEventListener(this.dropNode, 'dragover', this.handleDragOver);
-    addEventListener(this.dropNode, 'dragenter', this.handleDragEnter);
-    addEventListener(this.dropNode, 'dragleave', this.handleDragLeave);
+    addEventListener(dropNode, 'drop', this.handleDrop);
+    addEventListener(dropNode, 'dragover', this.handleDragOver);
+    addEventListener(dropNode, 'dragenter', this.handleDragEnter);
+    addEventListener(dropNode, 'dragleave', this.handleDragLeave);
     addEventListener(window, 'resize', this.adjustSize);
 
+    this.adjustSize();
```

Comment:
Previously this was being called every render now it's just being called in `componentDidMount`

Reply:
After some study on ref callbacks, I've understood that ref callbacks are called before the component mounts (if the component is rendered) and not on every render. So, with the new refs API, it would be best to call it in ComponentDidMount alone?